Makefile

```diff
@@ -149,13 +149,16 @@ extension-developer-e2e: run image-registry test-ext-dev-e2e kind-clean
 e2e-coverage:
 	COVERAGE_OUTPUT=./e2e-cover.out ./hack/e2e-coverage.sh
 
-.PHONY: kind-load
```

Comment:
Why remove the .PHONY target annotation here?  We don't generate a versioned artifact as a result of this make target.

Reply:
I believe this was an accidental keystroke, will fix and thanks for catching.

Reply:
ugh, I just blew up this PR with an ill-advised force push. For a small change like this I'm going to start over.